networkx/drawing/layout.py

```diff
@@ -351,6 +351,7 @@ def fruchterman_reingold_layout(
     iterations=50,
     threshold=1e-4,
     weight="weight",
+    node_weight=None,
```

Comment:
Inserting a parameter in the middle of the signature like this technically breaks backward compatibility, though I suspect most users aren't using only positional args. In principle, this can be avoided by moving the new parameter to the end of the list, though I certainly see the merit of keeping it near the `weight` parameter.

This is probably okay, though the API change should be mentioned in the release notes at least.

Reply:
True, for the reasons you stated I felt that adding it after `weight` was the best option, but I'm not especially attached to it so whatever you feel is best for the long-haul is fine. Another possibility is to allow weight to be a list or dict to include both weightings in the same parameter. If a single value is given then it's treated as the weighting for the edges.